wp-parsely.php

```diff
@@ -211,7 +211,9 @@ function maybe_load_plugin() {
 	}
 
 	// Require the actual wp-parsely plugin.
-	require_once $entry_file;
+	if ( is_readable( $entry_file ) ) {
+		require_once $entry_file;
+	}
```

Comment:
Maybe flip around? If not readable then return early, otherwise load and let the active versions and whatnot